pkg/csi/service/wcpguest/controller_helper.go

```diff
@@ -275,15 +276,49 @@ func isPVCInSupervisorClusterBound(ctx context.Context, client clientset.Interfa
 		if !ok {
 			continue
 		}
-		log.Debugf("PersistentVolumeClaim %s in namespace %s is in state %s. Received event %v",
-			pvcName, ns, pvc.Status.Phase, event)
-		if pvc.Status.Phase == v1.ClaimBound && pvc.Name == pvcName {
-			log.Infof("PersistentVolumeClaim %s in namespace %s is in state %s", pvcName, ns, pvc.Status.Phase)
-			return true, nil
+
+		if pvc.Name == supervisorPvcName {
+			if pvc.Status.Phase == v1.ClaimBound {
+				log.Infof("PersistentVolumeClaim %s in namespace %s is in state %s", supervisorPvcName, supervisorPvcNamespace, pvc.Status.Phase)
+				return true, nil
+			} else if pvc.Status.Phase == v1.ClaimPending {
+				var failureMessage string
+				_ = wait.PollImmediate(2*time.Second, 1*time.Minute, func() (done bool, err error) {
```

Comment:
Why do we need the `wait.PollImmediate`? What happens if the SV PVC gets bound within this 1 minute?